<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServerMaintenanceToolClass</class>
 <widget class="QMainWindow" name="ServerMaintenanceToolClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ServerMaintenanceTool</string>
  </property>
  <property name="windowIcon">
   <iconset resource="servermaintenancetool.qrc">
    <normaloff>:/ServerMaintenanceTool/Resources/LBA-net.ico</normaloff>:/ServerMaintenanceTool/Resources/LBA-net.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QGroupBox" name="groupBox_World">
      <property name="title">
       <string>World</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="l_WorldName">
         <property name="text">
          <string>World name:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="cb_WorldName"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="l_textstatus">
         <property name="text">
          <string>World server status:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="l_serverstatus">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_action">
      <property name="title">
       <string>Action</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="b_Update">
         <property name="text">
          <string>Update</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="b_Start">
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="b_Stop">
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_status">
      <property name="title">
       <string>Output</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>2</number>
       </property>
       <item>
        <widget class="QPlainTextEdit" name="plainTextEdit">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="servermaintenancetool.qrc"/>
 </resources>
 <connections/>
</ui>
